Documentation/metrics.md

```diff
@@ -115,4 +115,4 @@ Heavy file descriptor (`process_open_fds`) usage (i.e., near the process's file
 [prometheus-getting-started]: http://prometheus.io/docs/introduction/getting_started/
 [prometheus-naming]: http://prometheus.io/docs/practices/naming/
 [v2-http-metrics]: v2/metrics.md#http-requests
```

Comment:
@lucperkins since v2/metrics.md is removed, shouldn't this and related reference be removed from here? Thanks!